repository-data/application/src/main/resources/hcm-content/content/documents/administration/website.yaml

```diff
@@ -1,6 +1,67 @@
 ---
 /content/documents/administration/website:
   .meta:order-before: publication-system
+  /generic-labels:
```

Comment:
same thing as last time.

if you move "generic-labels" to it's own yaml file, it will be created automatically on next release.

When you leave it in the current form, you will have to manually login to all 3 environments and create these files manually.

Reply:
Good point, I'll have a look!